### DISCUSSION TITLE
相信党，相信政府。你是怕把你这个repo封掉，还是打心眼里真的这样想？

### DISCUSSION THREAD

**Discussion by pippoflow** [WeNeedHome/SummaryOfLoanSuspension#890] [General]
没别的意思，就是很想知道。谢谢

**Comment by Accelerator6666** [comment]
08年股灾，相信D相信政府的结局是什么样你自己可以去了解一下嘛

**Reply by repairbean** [reply]
16年棚改，涨价去库存呢

**Reply by zekijun** [reply]
具体点，信息太多了老哥，你直接告诉我们08年的措施和对结局的影响。

**Comment by baoang** [comment]
在Github发贴，不是应该写相信微软相信代码吗？

**Reply by WallStreetHu** [reply]
真相了

**Reply by 5EAR** [reply]
对强权的服从已经刻进骨子里了

**Comment by Susan9001** [comment]
唉，题主既然都这么写了，心里也清楚他是不可能正面作答的。既是如此，何必问呢。

**Comment by pippoflow** [comment]
我只是觉得，不“显式”地写这句，也不妨碍你在这开一个repo。而即使你写了，它想封还是会封。何苦呢？是真的还相信你声称你相信的？如果真的相信，我很想听听你们到底是怎么想的。交流一下而已。

**Reply by hrsgn** [reply]
是为了人身安全

**Comment by yanwuguangjun** [comment]
这句话 或许只是写出来 不是给你看的。

**Comment by computernot** [comment]
如果被请喝茶，这些话会有帮助。

**Comment by luowenfeng** [comment]
他要是真信了, 就该发在gitee上

**Comment by solidusxin** [comment]
贴毛泽东的话语，有时候就感觉很迷。到底是一种自我保护，还是投机取巧

**Reply by ztex** [reply]
总感觉是一种圣君情节 没得救

**Reply by BobbyYang9** [reply]
老百姓只是需要一个革命者来推翻既有制度和固化阶级，这个人是不是姓毛其实无所谓，现在老百姓没有任何办法去改变现状，只能祈祷革命者去代劳，而毛恰恰是离人们最近的革命者。能理解人们希望毛回来的心，并不是因为期望那个时代，如果全部回到计划经济，大锅饭时代大部分人还是要跳脚，人们只是盼着变天罢了，缅怀毛是最好的对当权者的愤慨，毕竟当权者也不敢因为你崇拜毛的革命而说三道四，但你自己喊出：我想革命的时候，基本上离死不远了。有点像古时候拿前太上皇骂当今皇上昏庸，当今皇上再想骂娘也不敢明目张胆去屠戮这些奴才[思考][思考][思考]

**Comment by zc0110** [comment]
你以为写了就是真信吗，懂什么叫反讽吗，包括7000人大会内容，细品

**Comment by clleo888** [comment]
這政府知道什麼是誠信嗎? 
好了, 明知道沒有誠信還相信的, 不是傻子是什麼?

**Comment by TALK1949** [comment]
有一说一，当下最好的避险方式就是别相信政府。
当然无论是有意还是无意，我都理解为什么要这么写，想在国内维权就得这样。

**Reply by solidusxin** [reply]
或者认为是一种魔法护盾，先给自己贴意识形态的标签，避免被人以境外势力，反华组织，图谋不轨来定义行为，避免落得寻衅滋事的罪名

**Reply by blue-ling233** [reply]
看看唐山的事情就知道，为啥别信了。所谓雷霆行动最后有啥

**Comment by miaodi** [comment]
当然是真心觉的，真心拥护习大大向天再借五百年

**Reply by oceanzhang860411** [reply]
五百年夠吃嗎？要不多克隆幾個習近平，可以永世統治

**Comment by ppinkiris** [comment]
恨国党能不能滚出这里，大家只想讨论最后怎么交房

**Reply by zc0110** [reply]
相信政府相信党，你很快就会拿到房了

**Reply by ppinkiris** [reply]
> 相信政府相信党，你很快就会拿到房了

我确实拿到了，还不止一套，有问题吗？

**Reply by zc0110** [reply]
没问题，拿到了就退群吧

**Comment by pippoflow** [comment]
楼上这个人，一来就贴标签，“恨国党”云云。。。别人礼貌讨论，突然来一句“滚出这里”。说实话，恶臭难当。
首先，党!=国；有人对“党”有意见、质疑，并不等于所谓“恨国”。你哪里被洗的脑，请你回哪里去。
其次，不要把“爱国”和“爱党”等同起来；不要觉得人家在政治上不认同一个政党，就是不爱自己的故土。所谓爱国，本就不应该整天放在台面上被翻炒，因为这只是基本人性而已。就像你爱你爸妈一样。无它，人性使然。
最后，请你清醒点，也请你把嘴巴放干净。
